### DISCUSSION TITLE
wind build limits

### DISCUSSION THREAD

**Discussion by ssattler** [NREL/ReEDS-2.0#54] [Q&A: Analysis]
We're running some high RE penetration scenarios for particular states and we are seeing some pretty big wind builds within each time step - bigger than we can reasonably expect happening. How can we limit the wind builds in a state to a set number of MW?
Thanks!

Also, my apologies for originally posting this in the wrong section of GitHub.

**Comment by vcelloho** [comment]
There exist constraints in the model that will limit the relative and total amount of new installations for a given technology. These constraints are currently formulated to apply at the CONUS level reflecting a limit to industry growth potential. My recommendation is to take these and index at the state level. There isn't currently an out of the box capability for this at the time, but I'd be happy to provide additional pointers for how to implement this.

Both constraints are located in c_supplymodel.gms

eq_growthlimit_relative

eq_growthlimit_absolute

**Reply by ssattler** [reply]
It would be great if you could provide additional guidance on how I can do this. I would appreciate the help.
